### PR TITLE
PT-3015: After removing an A&W node, the A&W label stays in the pedigree; PT-3032: A&W symbol and adoption brackets overlap for females

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

File: components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js [hunk 1]
```diff
@@ -1100,7 +1102,7 @@ define([
          */
         getAllGraphics: function($super) {
             //console.log("Node " + this.getNode().getID() + " getAllGraphics");
-            return $super().push(this.getHoverBox().getBackElements(), this.getLabels(), this._linkArea, this.getCarrierGraphics(), this.getEvaluationGraphics(), this.getHoverBox().getFrontElements());
+            return $super().push(this.getHoverBox().getBackElements(), this.getLabels(), this._awLabel, this._linkArea, this.getCarrierGraphics(), this.getEvaluationGraphics(), this.getHoverBox().getFrontElements());
```
Comment (allasm) [inline components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js]: This works here but is not right, as may cause other bugs. Instead, `this.getLabels()` should include `this._awLabel`. Other code may also expects all labels to be set in `getLabels()` (e.g. code to move labels under certain conditions)

Comment (allasm) [inline components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js]: This works here but is not right, as may cause other bugs. Instead, `this.getLabels()` should include `this._awLabel`. Other code may also expects all labels to be set in `getLabels()` (e.g. code to move labels under certain conditions)

**Comment by veronikaslc** [comment]
@allasm added `this._awLabel` to `this.getLabels()`

**Comment by veronikaslc** [comment]
Appeared that original fix was correct, so reverted it back

**Comment by marta-** [comment]
@sashaandjic please test.

**Comment by sashaandjic** [comment]
+1

**Pull request merged**